README.md

```diff
@@ -82,26 +114,39 @@ This action creator can take the following argument:
 |---|---|---|
 | `state` | `Object` | (required) current state of the your application |
 
-#### `setUserAgent`
+```js
+import { getCookies } from '@americanexpress/one-app-ducks';
+
+// ...
+
+const cookies = getCookies(store.getState());
+```
+
+#### Action Creators
+
+##### `setUserAgent`
 
-This action creator can be used to set the [user-Agent](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent).
+> ⚠️ For Internal Use
```

Comment:
What does this mean? Internal as in for use only by One App?

Reply:
yes